src/Components/Collections/Tree/TreeView/Thing.js

```diff
@@ -71,7 +71,13 @@ const Thing = ({child, collection, parent, thing, things, watchIds, inset}) => {
     };
 
     const handleOpenClose = (open) => {
-        open && thing && thing['#'] && CollectionActions.queryWithReturnDepth(collection.collection_id, collection.name, thing['#']);
+        if(open) {
+            if(thing && thing['#']) {
```

Comment:
Shouldn't the `thing && thing['#']` check by outside the `if (open)` check? since `thing['#']` is also used in the `else` case... If not, maybe add a comment why this is so?

Reply:
Updated in 43b8a4a.